packages/neso-particles/package.py

```diff
@@ -14,8 +14,8 @@ class NesoParticles(CMakePackage):
     version("0.4.0", commit="c615974661e0f4c8d9db709d65d27cc2927bbfaf")
     version("0.3.1", commit="9c6b4626645f6aaaca478e4798f2fdee5dd2675b")
     version("0.3.0", commit="cb55184bb7196c210d8c3f4397f4a29808acd038")
-    version("0.2.0", commit="2e1b1aac6f4f9c22b31787cf300fcc2b4914cbe1")
-    version("0.1.0", commit="bae6ee91fe6558a0d7eba040dd93db34eda348a8")
+    version("0.2.0", commit="bae6ee91fe6558a0d7eba040dd93db34eda348a8")
```

Comment:
@will-saunders-ukaea I got a conflict with the 0.1.0 and 0.2.0 hashes when I merged - could you quickly check and make sure they're correct?

Reply:
Main looks swapped, here looks correct - see the right hand column here: https://github.com/ExCALIBUR-NEPTUNE/NESO-Particles/releases